### PR TITLE
Slack Env comparison fix

### PR DESCRIPTION
**Pull request by ankit-privado**
Added Slack Token to workflow's ubuntu instance.

### CODE REVIEW

File: .github/workflows/comparison-results.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ jobs:
 
       # langauge specific repository file
       - name: Run the script for ${{github.head_ref}} and ${{github.base_ref}}
-        run: export GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }} && cd ./temp/standalone-monitoring-stability && pip install -r requirements.txt && python3 ./run.py -r ./repos/${{matrix.language}}.txt -b ${{github.base_ref}} -h ${{github.head_ref}} -bcr ${{ github.event.pull_request.base.repo.html_url }} -hcr ${{ github.event.pull_request.head.repo.html_url }} -guf
+        run: export GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }} && export SLACK_TOKEN=${{ secrets.SLACK_TOKEN }} && cd ./temp/standalone-monitoring-stability && pip install -r requirements.txt && python3 ./run.py -r ./repos/${{matrix.language}}.txt -b ${{github.base_ref}} -h ${{github.head_ref}} -bcr ${{ github.event.pull_request.base.repo.html_url }} -hcr ${{ github.event.pull_request.head.repo.html_url }} -guf
```
Comment (karan-batavia) [inline .github/workflows/comparison-results.yml:61]: Can we use the `env` key for this? It is much cleaner, and will help if we want to add more environment variables.

**Review by karan-batavia — CHANGES_REQUESTED** [review]

File: .github/workflows/comparison-results.yml (hunk 1 above)
Comment (ankit-privado) [reply .github/workflows/comparison-results.yml:61]: Yes right, need to change at multiple places like where its fetched on comparison repo.

Comment (karan-batavia) [reply .github/workflows/comparison-results.yml:61]: Yes, discovered this while working on the pipeline. Should be a small change.

**Review by HiteshMah-Jan — APPROVED** [review]

**Comment by ankit-privado** [comment]
As discussed with @karan-batavia , the fix for the issue mentioned above is not required now since we have already switched to using Privado's own Joern instead of the previously publicly available Joern. Therefore, im closing this PR for now.

**Pull request closed without merging**